tests/Microsoft.Identity.Test.LabInfrastructure/LabUser.cs

```diff
@@ -79,6 +79,22 @@ public class LabUser
         [JsonProperty("b2cProvider")]
         public B2CIdentityProvider B2CIdentityProvider { get; set; }
 
+
+        private string _password = null;
+
+        public string Password
```

Comment:
SecureString ?

Reply:
SecureString's main use case is when the user types in a password a char at a time. In an UI Test we are doing the opposite - we fetch the password from KeyValut (as a string) and we type it in the browser. SecureString isn't really meant for that.

In any case, [KeyVault only supports string](https://github.com/Azure/azure-sdk-for-net/issues/1819) so in any case the test is vulnerable to memory dump attack.

Reply:
I disagree with the main use case, it's absolutely not just for users typing in passwords in the ui, however if KeyVault doesn't support it then the code will not necessarily be improved by adding secure string.